aten/src/ATen/Context.cpp

```diff
@@ -17,45 +17,48 @@
 #if defined(__aarch64__) && !defined(C10_MOBILE)
 #include <cpuinfo.h>
 #endif
-
+#include <iostream>
 namespace at {
 
 static std::vector<std::string> generic_precisions = {"default", "tf32", "bf16"};
 static std::vector<std::string> cuda_precisions = {"default", "tf32"};
 static std::vector<std::string> mkldnn_precisions = {"default", "bf16"};
 
-// This method first check whether the backend and operators are legal
-// After that, if there is non-empty precision
-// it will futher check whether the precision is legal
-// and return the result
-static bool validate_fp32_prec_with_backend_and_op(
+// Check whether the backend and op are legal
+static void validate_fp32_prec_backend_and_op(
```

Comment:
nit: perhaps `check_fp32_prec_backend_and_op` is better since we are doing assertion checks here?

Reply:
Updated